tools/profileBuilder/program_test.go

```diff
@@ -92,3 +98,31 @@ func Test_getAliasPath(t *testing.T) {
 		})
 	}
 }
+
+func Test_generateAliasPackages(t *testing.T) {
+	files := token.NewFileSet()
+	gopath := strings.Replace(os.Getenv("GOPATH"), "\\", "/", -1)
+	folder := path.Join(gopath, "src", "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2015-06-15/network")
+	iterations := 10
+
+	packagesEnumerator := collection.NewList()
+	parsed, err := parser.ParseDir(files, folder, nil, 0)
+
+	if err != nil {
+		errLog.Printf("Couldn't open %q because: %v", folder, err)
+		return
+	}
+
+	for _, entry := range parsed {
+		packagesEnumerator.Add(entry)
+	}
+
+	want := generateAliasPackages(packagesEnumerator.Enumerate(nil)).ElementAt(0).(*alias).AliasPackage
```

Comment:
I'd rather have this be more of an end-to-end test, and instead of comparing at the model level, compare the text output that is spit into a file or text buffer.

Reply:
I would say we want to ensure that the model we build is always the same which we do, but unit testing that printing is always the same seems unnecessary.

Reply:
I'm okay agreeing to disagree.